### ISSUE TITLE
HBase init action fails on HA cluster with `enable-kerberos=true`

### ISSUE THREAD

**Issue by egor-doudkin-mpf** [GoogleCloudPlatform/dataproc-initialization-actions#588]
As stated in the description, Hbase init action does not work properly on HA dataproc cluster with Kerberos enabled and `enable-kerberos=true` init option.

The steps to reproduce is pretty straight-forward and consists of only one step

1. Create a minimalistic HA dataproc cluster with Kerberos and observe the results
```sh 
gcloud dataproc clusters create dataproc-cdap-ha-4 \
  --num-masters 3 \
  --initialization-actions gs://dataproc-initialization-actions/hbase/hbase.sh \
  --metadata 'enable-kerberos=true,keytab-bucket=<bucket-uri>' \
  --kerberos-root-principal-password-uri="<another-bucket-uri>" \
  --kerberos-kms-key="<kms-key-path>"
```

After some time, the logs on the master will show you that the init action cannot create Kerberos principals for secondary master because the `kadmin.local` tool gets invalid input arguments

```
+ kadmin.local -q 'addprinc -randkey hbase/dataproc-cdap-ha-4-m-1 dataproc-cdap-ha-4-m-2.c.mp-the-rock-dwayne-johnson.internal@C.MP-THE-ROCK-DWAYNE-JOHNSON.INTERNAL'
usage: add_principal [options] principal
	options are:
		[-randkey|-nokey] [-x db_princ_args]* [-expire expdate] [-pwexpire pwexpdate] [-maxlife maxtixlife]
		[-kvno kvno] [-policy policy] [-clearpolicy]
		[-pw password] [-maxrenewlife maxrenewlife]
		[-e keysaltlist]
		[{+|-}attribute]
	attributes are:
		allow_postdated allow_forwardable allow_tgs_req allow_renewable
		allow_proxiable allow_dup_skey allow_tix requires_preauth
		requires_hwauth needchange allow_svr password_changing_service
		ok_as_delegate ok_to_auth_as_delegate no_auth_data_required
		lockdown_keys

where,
	[-x db_princ_args]* - any number of database specific arguments.
			Look at each database documentation for supported arguments
```

**Comment by egor-doudkin-mpf** [comment]
Hey, I have created a PR to address this - https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/594

**Comment by medb** [comment]
@cyxxy Should we add test case for this, to not break it again?

**Comment by medb** [comment]
Fixed in #594
